main.js

```diff
@@ -0,0 +1,9 @@
+const open = document.querySelector('.menu__icon');
+open.addEventListener('click', () => {
+  document.getElementById('slide_menu').classList.add('active');
+});
+
+const close = document.querySelector('#slide_menu');
+close.addEventListener('click', () => {
+  document.getElementById('slide_menu').classList.remove('active');
+});
```

Comment:
When then menu bar is clicked its functioning ok but the problem is the blue shadowed background isnt fitting the entire mobile screen. Kindly fix this as well.

Your design:
![Screenshot from 2022-05-12 20-23-18](https://user-images.githubusercontent.com/79506105/168126962-382e6b67-f1b0-4f99-974c-a74305ce90a2.png)